examples/deployment/README.md

```diff
@@ -0,0 +1,6 @@
+ASA deployment using Algorand builder from Scale it
+
+Needs to be linked (`yarn link`) with algob:
+https://github.com/scale-it/algorand-builder/
+
+Authors: Invertisment & amityadav0
```

Comment:
this is a whole team effort, but you can link your repository if you are planning to do more updates.

Reply:
I think it's good to have it in a repository but I'm not sure how to handle the updates then. 
I can mention it. 
I'll change this line.

Reply:
I'll add all 4 people.